### DISCUSSION TITLE
Data Structures

### DISCUSSION THREAD

**Discussion by rwl7532** [jkotlinski/durexforth#547] [Q&A] [answered]
Library of Forth routines and utilities by Terry, James has a section on Data Structures with various words defined.
: 1array.  create
here swap 2* dup allot erase does> swap 2* ;
erase in durexforth is already defined for graphics.
What word and it's definition would be the equivalent?

**Comment by jkotlinski** [comment]
Hi!
Standard ERASE was actually added to master just a while ago.
It is a mistake that gfx has an ERASE, too, it should be renamed to
something else. (Any suggestions?)

fre 21 apr. 2023 kl. 05:12 skrev rwl7532 ***@***.***>:

**Reply by rwl7532** [reply]
How do I  update my  .d64 file that I'm using with my C64? (by SD card)

**Reply by jkotlinski** [reply]
You can get a new build from master branch here : https://github.com/jkotlinski/durexforth/suites/12360359185/artifacts/656387429

**Comment by jkotlinski** [comment]
Maybe gfx ERASE could be XORPLOT…

fre 21 apr. 2023 kl. 07:40 skrev Johan Kotlinski ***@***.***>:

**Reply by rwl7532** [reply]
SCRUB would suffice I would think.

**Reply by jkotlinski** [reply]
RUB?

**Reply by rwl7532** [reply]
RUBOUT ?

**Reply by jkotlinski** [reply, marked as answer]
PEN?

**Reply by rwl7532** [reply]
PEN is good as 0 or 1 determines the mode of making a line (I think)

**Reply by jkotlinski** [reply]
OK, consider it renamed.
